packages/node_modules/@webex/plugin-logger/src/logger.js

```diff
@@ -194,7 +194,7 @@ const Logger = WebexPlugin.extend({
    * @returns {boolean}
    */
   shouldBuffer(level) {
-    return precedence[level] <= precedence.info;
+    return precedence[level] <= (this.config.bufferLogLevel ? precedence[this.config.bufferLogLevel] : precedence.info);
```

Comment:
Is `bufferLogLevel` a new configuration attribute?

Reply:
@glenatwork once you answer this question and resolve the conversation i can move forward with a merge

Reply:
@lalli-flores yes bufferLogLevel is a new config setting so as to not interfere with behavior of any existing setting (ie keep everything backwards compatible).  also this way the level can be different for the console vs. the log file.
@taymoork2 hope that answers the question.  thanks for the help.

Reply:
@glenatwork since the change is small, can you split these commits to be a `feat()` and a `test()` and for me it should be good to merge

Reply:
@taymoork2 done!